Docker.md

```diff
@@ -28,7 +29,8 @@ services:
 
 #### Docker run
 
-```bash
```

Comment:
This should stay this way. It is how you tell GitHub flavored markdown what language to highlight the code in the code block as.

Reply:
before the \```bash line disappeared so I adjusted to the `bash` was visible.
change noted.